ui/components/app/token-list-display/token-list-display.scss

```
@@ -0,0 +1,43 @@
+.loading-span {
+  display: flex;
+  height: 250px;
+  align-items: center;
+  justify-content: center;
+  padding: 30px;
```

Adhere to `8px` grid
```suggestion
  padding: 32px;
```